src/index.js

```diff
@@ -1,10 +1,14 @@
+import Vue from 'vue';
+import Vuetify from 'vuetify';
```

Comment:
Any reason that you're using the global import? If you don't import from `vuetify/lib` you will end up getting every Vuetify component compiled.

Reply:
Not a particular reeason, but thinking of it right now i remember about 30-40 components that we're using.
(i saw it in the errors when i removed the loader)

Perhaps something for a new issue?